.tekton/scanner-pull-request.yaml

```
@@ -0,0 +1,446 @@
+      when:
+      - input: $(tasks.init.results.build)
+        operator: in
+        values:
+        - "true"
+      workspaces:
+      - name: output
+        workspace: workspace
+      - name: basic-auth
+        workspace: git-auth
+    - name: prefetch-dependencies
+      params:
+      - name: input
+        value: $(params.prefetch-input)
+      runAfter:
+      - clone-repository
+      taskRef:
+        params:
+        - name: name
+          value: prefetch-dependencies
+        - name: bundle
+          value: quay.io/redhat-appstudio-tekton-catalog/task-prefetch-dependencies:0.1@sha256:0b7bec23b6c08f37138a86e569835842763b3aa42f4455fd70ba3986350e07c7
+        - name: kind
+          value: task
+        resolver: bundles
+      when:
+      - input: $(params.hermetic)
+        operator: in
+        values:
+        - "true"
+      workspaces:
+      - name: source
+        workspace: workspace
+    - name: fetch-vuln-feed-data
+      runAfter:
+      - init
+      - clone-repository
+      taskSpec:
+        steps:
+          - name: fetch-vuln-feed-data
+            image: registry.access.redhat.com/ubi8/ubi
+            script: |
+              #!/usr/bin/env bash
+              mkdir -p "$(workspaces.source.path)/source"
+              blobs=(
+                nvd-definitions.zip
+                k8s-definitions.zip
+                repo2cpe.zip
+                genesis_manifests.json
+              )
+              for blob in "${blobs[@]}"; do
+                echo "https://storage.googleapis.com/definitions.stackrox.io/scanner-data/latest/${blob} > $(workspaces.source.path)/source/blob-${blob}"
+                curl --fail -s --show-error --retry 4 --retry-max-time 30 --retry-connrefused \
+                  --output "$(workspaces.source.path)/source/blob-${blob}" \
+                  "https://storage.googleapis.com/definitions.stackrox.io/scanner-data/latest/${blob}"
+
+                if [ "$?" != "0" ]; then
+                    echo "Failed to download"
+                    exit 1
+                fi
+
+                ls -lh $(workspaces.source.path)/source
+              done
+      workspaces:
+      - name: source
+        workspace: workspace
+    - name: build-container
+      params:
+      - name: IMAGE
+        value: $(params.output-image)
+      - name: DOCKERFILE
+        value: $(params.dockerfile)
+      - name: CONTEXT
+        value: $(params.path-context)
+      - name: HERMETIC
+        value: $(params.hermetic)
+      - name: PREFETCH_INPUT
+        value: $(params.prefetch-input)
+      - name: IMAGE_EXPIRES_AFTER
+        value: $(params.image-expires-after)
+      - name: COMMIT_SHA
+        value: $(tasks.clone-repository.results.commit)
+      runAfter:
+      - prefetch-dependencies
+      - fetch-vuln-feed-data
+      taskRef:
+        params:
+        - name: name
+          value: buildah-6gb
+        - name: bundle
+          value: quay.io/redhat-appstudio-tekton-catalog/task-buildah-6gb:0.1
```

I think that the system may require to pin a digest for this image. I usually find those in the output of `docker pull quay.io/redhat-appstudio-tekton-catalog/task-buildah-6gb:0.1`.

```suggestion
          value: quay.io/redhat-appstudio-tekton-catalog/task-buildah-6gb:0.1@sha256:9c9b4081d873a217f0e20bcc7be3cd2ff96887f4c989439544640e3d2bc0e8b9
```

---

I can't remember why I switched to the 6gb image, but I vaguely remember identifying something that was taking up memory and somehow removing it.  So I just tried to switch back to the standard buildah image, and it worked just fine :shrug: (ffe1f37)